app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -15,6 +15,10 @@ navigation_description: Learn more about teacher training fees in England and fi
 date: "2021-05-27"
 image: "static/content/hero-images/0002.jpg"
 backlink: "../../"
+inset_text:
+  funding-holding-message:
+    text: |-
+      The following amounts apply if you're starting your teacher training course between September 2023 and July 2024. Financial support for courses starting the following year will be announced in autumn 2023.
```

Comment:
@EllieNodder Is this slightly confusing - the page talks about IRP, fees, loans etc. as well as bursaries and scholarships?

This banner implies that they are all being updated in autumn 2023 which I don't think is the case?

Should we be a bit more explicit?

```suggestion
      The bursary and scholarship amounts on this page apply if you're starting your teacher training course between September 2023 and July 2024. Financial support for courses starting the following year will be announced in autumn 2023.
```